app/assets/stylesheets/utilities/_background.scss

```diff
@@ -1,21 +1,16 @@
-.bg-gray-lighter {
-  background-color: $gray-lighter;
-}
-.bg-light-blue {
-  background-color: $blue-light;
-}
-.bg-lightest-blue {
-  background-color: $blue-lightest;
-}
-
-@media #{$breakpoint-sm} {
-  .sm-bg-light-blue {
-    background-color: $blue-light;
+@include at-media('tablet') {
+  /* stylelint-disable-next-line selector-class-pattern */
+  .tablet\:bg-primary-lighter {
```

Comment:
The disable here is because the selector technically doesn't conform to [the enforced BEM pattern](https://github.com/18F/identity-idp/blob/b95fabd1ea96480ebfb8de5150598f1f6115fa27/app/javascript/packages/stylelint-config/index.js#L11).

A few options were considered:

- Expand the pattern to allow the responsive prefix, either at the base Stylelint config, or in a project-specific override
- Override USWDS settings to generate responsive variants for background colors ([disabled by default](https://github.com/uswds/uswds/blob/d9d4b91233aa8e650a32107fd58cde47278589ac/src/stylesheets/theme/_uswds-theme-utilities.scss#L112))
- Refactor markup to avoid responsive variations for background color

I think the last of these could be a good thing to consider long-term, especially considering footer usage toward the [Identifier component](https://designsystem.digital.gov/components/identifier/). In the meantime, I think it's reasonable to keep these around, at least named in a way consistent with how one would expect to use responsive variations for utility classes.